public/API/API_GetUserSummary.php

```diff
@@ -121,9 +121,9 @@
     unset($retVal['LastGame']['RichPresencePatch']);
     unset($retVal['LastGame']['system']);
 } elseif ($recentGamesPlayed === 0) {
-    // if no games were requested, initialize empty arrays for Awarded and RecentAchievements
-    $retVal['Awarded'] = [];
-    $retVal['RecentAchievements'] = [];
+    // if no games were requested, initialize empty objects for Awarded and RecentAchievements
+    $retVal['Awarded'] = json_encode(new stdClass());
+    $retVal['RecentAchievements'] = json_encode(new stdClass());
```

Comment:
This returns strings:

```
GET http://localhost:64000/API/API_GetUserSummary.php?u=WCopeland&g=0&z=WCopeland&y=...
```

```json
	"Awarded": "{}",
	"RecentAchievements": "{}",
```

Reply:
This is hilarious and dumb, I didn't realize, thank you for catching this. Will fix.

Reply:
Consider using `ArrayObject` as was done for #484: https://github.com/Jamiras/RAWeb/blob/fc2e030c13e460930d47769d9807e125dbb3b123/public/API/API_GetGameInfoAndUserProgress.php#L59-L61

Reply:
Thank you very much, that seems to have done it! Still picking up some of the quirks of PHP!